ingest/src/main/java/com/microsoft/azure/kusto/ingest/AzureStorageClient.java

```diff
@@ -86,8 +86,8 @@ void uploadFileToBlob(File sourceFile, BlobClient blobClient) throws IOException
         blobClient.uploadFromFile(sourceFile.getPath());
     }
 
-    void uploadStreamToBlob(InputStream inputStream, String blobName, BlobContainerClient container, boolean shouldCompress)
-            throws IOException, URISyntaxException, BlobStorageException {
+    int uploadStreamToBlob(InputStream inputStream, String blobName, BlobContainerClient container, boolean shouldCompress)
```

Comment:
It's not clear that these methods return the size of the original stream, and only one is commented. 
I guess it's not that bad if they're internal, but it should probably be documented